src/CMakeLists.txt

```diff
@@ -57,6 +60,7 @@ set(freerct_SRCS ${freerct_SRCS}
     "${CMAKE_SOURCE_DIR}/src/generated/coasters_strings.h"
     "${CMAKE_SOURCE_DIR}/src/generated/gui_strings.h"
     "${CMAKE_SOURCE_DIR}/src/generated/shops_strings.h"
+    "${CMAKE_SOURCE_DIR}/src/generated/comp_info.h"
```

Comment:
rev implies reversion to me so it could  make  more sense to put them both together in compiling_information.h?